src/include/optimizer/cost_and_stats_calculator.h

```diff
@@ -35,7 +35,8 @@ class CostAndStatsCalculator : public OperatorVisitor {
       std::vector<double> child_costs);
 
   inline std::shared_ptr<Stats> GetOutputStats() {
-    return std::move(output_stats_);
+//    return std::move(output_stats_);
```

Comment:
Any reason why we want to change this to shared_ptr?

Reply:
Yes, because in the Cost we use shared_ptr for output_stats, if I cast unique_ptr to shared_ptr to call the functions in Cost, there'll be some issues when destroying pointers.